cfgov/jinja2/v1/_includes/atoms/large_checkbox.html

```diff
@@ -0,0 +1,57 @@
+{# ==========================================================================
+
+   large_checkbox.render()
+
+   ==========================================================================
+
+   Description: Builds checkboxes fields with a large hit target.
+
+   settings (required):
+
+   settings.legend:     Name of the field.
+
+   settings.fields:     An array of all the checkbox labels.
+
+   settings.size:       The size of the field. Defaults to 1-2.
+                        Possible options: 1, 1-2, 1-4
```

Comment:
Theoretically you are right @anselmbradford but from an interface design POV I don't anticipate using those super sized checkbox/radio button targets in a filter setting.